.github/workflows/ci.yml

```
@@ -13,8 +13,8 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-20.04]
-        python-version: ['3.8', '3.12']
+        os: [ubuntu-latest]
```

Let's keep the version to fixed notation so we can update it along with all other workflows and know which version is currently being used.
```suggestion
        os: [ubuntu-20.04]
```